gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java

```diff
@@ -342,6 +348,15 @@ public void orchestrate(Spec spec, Properties jobProps, long triggerTimestampMil
     Instrumented.updateTimer(this.flowOrchestrationTimer, System.nanoTime() - startTime, TimeUnit.NANOSECONDS);
   }
 
+  /*
+  Deletes spec from flowCatalog if it is an adhoc flow (not containing a job schedule)
+   */
+  private void deleteSpecFromCatalogIfAdhoc(FlowSpec flowSpec) {
```

Comment:
nit: `private` methods like this belong beneath the `public` and `protected` ones